docs/usage/administration/database_maintenance_tools.md

```diff
@@ -1,4 +1,5 @@
-This blog post by Victor Berger explains how to use many of the tools listed on this page: https://levans.fr/shrink-synapse-database.html
+This blog post by Jackson Chen explains how to use many of the tools listed on this page: https://jacksonchen666.com/posts/2022-12-03/14-33-00/
+This blog post by Victor Berger is from 2020 & outdated, so please only use as light reference https://levans.fr/shrink-synapse-database.html
```

Comment:
```suggestion
This blog post by Victor Berger is from 2020 & outdated, so please only use as light reference: https://levans.fr/shrink-synapse-database.html```

Can you describe how Berger's post did not work for you?

Reply:
I don't know what a newsfile is, but can add one when I understand what it is.

Reply:
A newsfile is the same thing as a "changelog file". See the checklist in the PR description for more information.

Reply:
This PR contains a newsfile which also explains why this PR is being created in the first place.

Reply:
@DMRobertson do I need to add any additional files to this PR?